includes/FormFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -218,10 +218,16 @@ private static function buildDescriptorCore(
 				$config->get( ConfigNames::DatabaseClustersInactive )
 			);
 
+			// Deprecated usage
```

Comment:
Deprecated? Was this added by mistake?

Reply:
Well no it's the old config method, for backward compatibility, new method is config must be a key value pair, so the old method is just deprecated.